client/src/main/scala/drt/client/components/FlightChartComponent.scala

```diff
@@ -45,54 +51,69 @@ object FlightChartComponent {
 
+                      val toggleShowAllNationalities = (e: ReactEventFromInput) => {
+                        val newValue: Boolean = e.target.checked
+                        scope.modState(_.copy(showAllNationalities = newValue))
+                      }
+
+                      val (chartWidth, chartHeight): (Int, Int) = if (info.nationalities.size > 10 && state.showAllNationalities)
+                        (300 + 5 * info.nationalities.size, 300)
+                      else (300, 300)
+
                       val paxTypeData: ChartJsData = ChartJsData(sortedPaxTypes.map {
                         case (pt, _) => PaxTypes.displayNameShort(pt)
                       }, sortedPaxTypes.map(_._2.toDouble), "Live API")
-                      <.div(^.cls := "row",
-                        if (sortedNats.toMap.values.sum > 0)
-                          <.div(^.cls := "col-sm arrivals__table__flight__chart-box__chart nationality-chart",
-                            ChartJSComponent.Bar(
-                              ChartJsProps(
-                                data = nationalityData,
-                                300,
-                                300,
-                                options = ChartJsOptions.withSuggestedMax("Nationality breakdown", sortedNats.toMap.values.max + 5)
-                              )
-                            ))
-                        else
-                          EmptyVdom,
-                        if (sortedPaxTypes.toMap.values.sum > 0 && sortedAges.toMap.values.sum > 0)
-                          <.div(^.cls := "col-sm arrivals__table__flight__chart-box__chart passenger-type-chart",
-                            ChartJSComponent.Bar(
-                              ChartJsProps(
-                                data = paxTypeData,
-                                300,
-                                300,
-                                options = ChartJsOptions.withSuggestedMax("Passenger types", sortedPaxTypes.toMap.values.max + 5)
-                              )))
-                        else
-                          EmptyVdom,
-                        if (sortedAges.toMap.values.sum > 0)
-                          <.div(^.cls := "col-sm arrivals__table__flight__chart-box__chart age-breakdown-chart",
-                            ChartJSComponent.Bar(
-                              ChartJsProps(
-                                data = ageData,
-                                300,
-                                300,
-                                options = ChartJsOptions.withSuggestedMax("Age breakdown", sortedAges.toMap.values.max + 5)
+
+                      <.div(^.cls := "container arrivals__table__flight__chart-box", ^.width := (chartWidth * 3).toString + "px",
+                        <.div(^.className := "arrivals__table__flight__chart__show__count_div", sortedNats.toMap.values.sum, <.span(^.className := s"arrivals__table__flight__chart__show__nationalities",
```

Comment:
I think this is left over from debugging, right?

Reply:
It is not left over , it is changes for show all Nationalities and all css align to look nice

Reply:
I meant, you're printing `sortedNats.toMap.values.sum` in the top left of the popup. I'm pretty sure we don't want that there, right?

Reply:
I don't know how I got that and assumed it was always there . I will remove that .